components/dashboard/SentimentTracker/SentimentTracker.js

```diff
@@ -61,13 +61,15 @@ EmojiButton.propTypes = {
 };
 
 const SentimentTracker = props => {
-  const { onRecord, onClose, user, isComplete } = props;
+  const { onRecord, onClose, record, isComplete } = props;
   const { userDocRef } = useAuth();
   const [complete, setComplete] = useState(isComplete);
+  const [value, setValue] = useState(record);
```

Comment:
It seems like we might not need to maintain state here -- when the logged label changes, the `user` object will change, so we should be able to just rely on the passed in `record` value, right?

Reply:
(in other words, this is duplicating that state that the `user` object maintains, no?)

Reply:
initially add this and the [complete, setComplete] because of concern for lagging, but it doesn't seem to be an issue. Removed both states.